src/index.ts

```diff
@@ -16,11 +16,15 @@ export interface NotarizeAppOptions {
   appBundleId: string;
 }
 
+export interface TransporterOptions {
+  ascProvider: string;
```

Comment:
This parameter is optional so it should be marked with a `?`
```suggestion
  ascProvider?: string;
```